NE-2075: Add dynamic commit for aws-load-balancer-controller

Thealisyed opened the pull request:
Added the FULL_COMMIT  to the
tekton pipeline and Containerfile in order
to have dynamic commit for the aws-load-balancer-controller


### Description

Added the FULL_COMMIT  to the
tekton pipeline and Containerfile in order
to have dynamic commit for the aws-load-balancer-controller

### Checklist
- [NA ] Added tests that cover your change (if possible)
- [NA ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [NA ] Manually tested
- [X ] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [NA ] Backfilled missing tests for code in same general area :tada:
- [NA ] Refactored something and made the world a better place :star2:

alebedev87 commented:
/assign

alebedev87 commented:
@Thealisyed : can you please add a comment with which labels we had in the latest CPaaS build so that I can compare? Plus, since it's the downstream repository for the upstream k8s project, we should follow [the commit message format](https://github.com/openshift/aws-load-balancer-controller/blob/main/rebase.md#message-format-for-cherry-picked-commits).

Thealisyed commented:
Konflux [build](https://konflux-ui.apps.kflux-prd-rh03.nnv1.p1.openshiftapps.com/ns/aws-load-balancer-operator-tenant/applications/aws-lb-optr-1-3-rhel-9/pipelineruns/aws-load-balancer-controller-container-aws-lb-optr-1-3-rheg8t65):
```
{
  "architecture": "x86_64",
  "build-date": "2025-10-03T17:15:43Z",
  "com.redhat.component": "aws-load-balancer-controller-container",
  "com.redhat.delivery.backport:": "false",
  "com.redhat.license_terms": "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI",
  "commit": "d833b2c551682b970fb88288a0a5d3dc09a0670c",
  "cpe": "cpe:/o:redhat:rhel_e4s:8.6::baseos",
  "description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
  "distribution-scope": "public",
  "io.buildah.version": "1.41.4",
  "io.k8s.description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
  "io.k8s.display-name": "Red Hat Universal Base Image 9",
  "io.openshift.expose-services": "",
  "io.openshift.tags": "base rhel9",
  "maintainer": "Red Hat, Inc.",
  "name": "aws-load-balancer-controller",
  "org.opencontainers.image.revision": "d833b2c551682b970fb88288a0a5d3dc09a0670c",
  "org.opencontainers.image.source": "https://github.com/Thealisyed/aws-load-balancer-controller",
  "quay.expires-after": "5d",
  "release": "1755678605",
  "summary": "Provides the latest release of Red Hat Universal Base Image 9.",
  "url": "https://catalog.redhat.com/en/search?searchType=containers",
  "vcs-ref": "d833b2c551682b970fb88288a0a5d3dc09a0670c",
  "vcs-type": "git",
  "vendor": "Red Hat, Inc.",
  "version": "1.2.0"
}
```

cPaaS [build](https://brewweb.engineering.redhat.com/brew/archiveinfo?archiveID=16399323):
```
'Labels': {'architecture': 'x86_64',
                                             'build-date': '2025-09-29T10:06:57',
                                             'com.redhat.component': 'aws-load-balancer-controller-container',
                                             'com.redhat.license_terms': 'https://www.redhat.com/agreements',
                                             'commit': '81267cb451d197df2bb091a669b098484245a342',
                                             'cpe': 'cpe:/o:redhat:rhel_e4s:8.6::baseos',
                                             'description': 'The Extended Life '
                                                            'Base Image is '
                                                            'designed and '
                                                            'engineered to be '
                                                            'the base layer '
                                                            'for Red Hat '
                                                            'extended life '
                                                            'products. This '
                                                            'base image is '
                                                            'only supported '
                                                            'for approved Red '
                                                            'Hat products. '
                                                            'This image is '
                                                            'maintained by Red '
                                                            'Hat and updated '
                                                            'regularly.',
                                             'distribution-scope': 'public',
                                             'io.buildah.version': '1.33.12',
                                             'io.k8s.description': 'The '
                                                                   'Extended '
                                                                   'Life Base '
                                                                   'Image is '
                                                                   'designed '
                                                                   'and '
                                                                   'engineered '
                                                                   'to be the '
                                                                   'base layer '
                                                                   'for all of '
                                                                   'your '
                                                                   'containerized '
                                                                   'applications, '
                                                                   'middleware '
                                                                   'and '
                                                                   'utilities. '
                                                                   'This base '
                                                                   'image is '
                                                                   'freely '
                                                                   'redistributable, '
                                                                   'but Red '
                                                                   'Hat only '
                                                                   'supports '
                                                                   'Red Hat '
                                                                   'technologies '
                                                                   'through '
                                                                   'subscriptions '
                                                                   'for Red '
                                                                   'Hat '
                                                                   'products. '
                                                                   'This image '
                                                                   'is '
                                                                   'maintained '
                                                                   'by Red Hat '
                                                                   'and '
                                                                   'updated '
                                                                   'regularly.',
                                             'io.k8s.display-name': 'Red Hat '
                                                                    'Extended '
                                                                    'Life Base '
                                                                    'Image',
                                             'io.openshift.expose-services': '',
                                             'io.openshift.tags': 'base rhel8',
                                             'maintainer': 'Red Hat, Inc.',
                                             'name': 'aws-load-balancer-controller',
                                             'org.opencontainers.image.revision': '8a6ee3971416146a11dac391b1a54f0dd69adc17',
                                             'release': '27',
                                             'summary': 'Provides the latest '
                                                        'release of the Red '
                                                        'Hat Extended Life '
                                                        'Base Image.',
                                             'url': 'https://access.redhat.com/containers/#/registry.access.redhat.com/aws-load-balancer-controller/images/1.2.0-27',
                                             'vcs-ref': '339680fd6b6d139d7fabf2008d22b8a6b14b4f7f',
                                             'vcs-type': 'git',
                                             'vendor': 'Red Hat, Inc.',
                                             'version': '1.2.0'},
```

Thealisyed commented:
cPaaS build:
```
'Labels': {'architecture': 'x86_64',
                                             'build-date': '2025-09-29T10:06:57',
                                             'com.redhat.component': 'aws-load-balancer-controller-container',
                                             'com.redhat.license_terms': 'https://www.redhat.com/agreements',
                                             'commit': '81267cb451d197df2bb091a669b098484245a342',
                                             'cpe': 'cpe:/o:redhat:rhel_e4s:8.6::baseos',
                                             'description': 'The Extended Life '
                                                            'Base Image is '
                                                            'designed and '
                                                            'engineered to be '
                                                            'the base layer '
                                                            'for Red Hat '
                                                            'extended life '
                                                            'products. This '
                                                            'base image is '
                                                            'only supported '
                                                            'for approved Red '
                                                            'Hat products. '
                                                            'This image is '
                                                            'maintained by Red '
                                                            'Hat and updated '
                                                            'regularly.',
                                             'distribution-scope': 'public',
                                             'io.buildah.version': '1.33.12',
                                             'io.k8s.description': 'The '
                                                                   'Extended '
                                                                   'Life Base '
                                                                   'Image is '
                                                                   'designed '
                                                                   'and '
                                                                   'engineered '
                                                                   'to be the '
                                                                   'base layer '
                                                                   'for all of '
                                                                   'your '
                                                                   'containerized '
                                                                   'applications, '
                                                                   'middleware '
                                                                   'and '
                                                                   'utilities. '
                                                                   'This base '
                                                                   'image is '
                                                                   'freely '
                                                                   'redistributable, '
                                                                   'but Red '
                                                                   'Hat only '
                                                                   'supports '
                                                                   'Red Hat '
                                                                   'technologies '
                                                                   'through '
                                                                   'subscriptions '
                                                                   'for Red '
                                                                   'Hat '
                                                                   'products. '
                                                                   'This image '
                                                                   'is '
                                                                   'maintained '
                                                                   'by Red Hat '
                                                                   'and '
                                                                   'updated '
                                                                   'regularly.',
                                             'io.k8s.display-name': 'Red Hat '
                                                                    'Extended '
                                                                    'Life Base '
                                                                    'Image',
                                             'io.openshift.expose-services': '',
                                             'io.openshift.tags': 'base rhel8',
                                             'maintainer': 'Red Hat, Inc.',
                                             'name': 'aws-load-balancer-controller',
                                             'org.opencontainers.image.revision': '8a6ee3971416146a11dac391b1a54f0dd69adc17',
                                             'release': '27',
                                             'summary': 'Provides the latest '
                                                        'release of the Red '
                                                        'Hat Extended Life '
                                                        'Base Image.',
                                             'url': 'https://access.redhat.com/containers/#/registry.access.redhat.com/aws-load-balancer-controller/images/1.2.0-27',
                                             'vcs-ref': '339680fd6b6d139d7fabf2008d22b8a6b14b4f7f',
                                             'vcs-type': 'git',
                                             'vendor': 'Red Hat, Inc.',
                                             'version': '1.2.0'},
```

Konflux build:

```
  "architecture": "x86_64",
  "build-date": "2025-10-06T09:43:00Z",
  "com.redhat.component": "aws-load-balancer-controller-container",
  "com.redhat.license_terms": "https://www.redhat.com/en/about/red-hat-end-user-license-agreements#UBI",
  "commit": "cd7de65ba45cf3b116fb5f2a0fe2717645faea2d",
  "description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
  "distribution-scope": "public",
  "io.buildah.version": "1.41.4",
  "io.k8s.description": "The Universal Base Image is designed and engineered to be the base layer for all of your containerized applications, middleware and utilities. This base image is freely redistributable, but Red Hat only supports Red Hat technologies through subscriptions for Red Hat products. This image is maintained by Red Hat and updated regularly.",
  "io.k8s.display-name": "Red Hat Universal Base Image 9",
  "io.openshift.expose-services": "",
  "io.openshift.tags": "base rhel9",
  "maintainer": "Red Hat, Inc.",
  "name": "aws-load-balancer-controller",
  "org.opencontainers.image.revision": "cd7de65ba45cf3b116fb5f2a0fe2717645faea2d",
  "org.opencontainers.image.source": "https://github.com/Thealisyed/aws-load-balancer-controller",
  "quay.expires-after": "5d",
  "release": "1755678605",
  "summary": "Provides the latest release of Red Hat Universal Base Image 9.",
  "url": "https://catalog.redhat.com/en/search?searchType=containers",
  "vcs-ref": "cd7de65ba45cf3b116fb5f2a0fe2717645faea2d",
  "vcs-type": "git",
  "vendor": "Red Hat, Inc.",
  "version": "1.3.0"
}

```

alebedev87 reviewed (commented):
/lgtm
/approve
